### PR TITLE
Apply new design for Extended Canvas

### PR DESCRIPTION
**Pull request by joyl1216**
Screenshot_1604956949
Screenshot_1604956961
Screenshot_1604956969
Screenshot_1604956974

### CODE REVIEW

**Comment by Mehul1995** [comment]
Just realized; @joyl1216, is it possible for you to remove the hamburger menu from the app? Since there isn't any functionality on the app. Same with the share and search icon.

**Comment by Mehul1995** [comment]
This looks good; But you may choose to leave it as it is if you already have the activity for the menu functionality. But you can remove search and share icon from the app header.

**Review by Mehul1995 — APPROVED** [review]

**Comment by joyl1216** [comment]
Update the design with WebView with Google Map. Seems that we can't remove the side menu, but I covered the search bar with the tool bar. 

Screenshot_1605565558
Screenshot_1605565573
Screenshot_1605565582
Screenshot_1605565586

**Comment by Mehul1995** [comment]
Looks good to me!! Awesome work @joyl1216 👍

File: ExtendCanvas/src/main/java/com/microsoft/device/display/samples/extendcanvas/MainActivity.kt [hunk 1]
```diff
@@ -19,21 +16,11 @@ class MainActivity : AppCompatActivity() {
...
-        val searchView = menu.findItem(R.id.action_search).actionView as SearchView
-        if (searchManager.getSearchableInfo(componentName) != null) {
-            searchView.setSearchableInfo(searchManager.getSearchableInfo(componentName))
-        }
-        searchView.isIconifiedByDefault = false
-        searchView.isFocusable = true
-        searchView.requestFocusFromTouch()
-        searchView.onActionViewExpanded()
-        return true
+        val webView = findViewById<View>(R.id.webView) as WebView
```
Comment (CesarValiente) [inline ExtendCanvas/src/main/java/com/microsoft/device/display/samples/extendcanvas/MainActivity.kt:22]: I believe you can do:
```
val webView = findViewById<WebView>(R.id.webView)
```
So you can get rid of the cast at the end.

**Review by CesarValiente — APPROVED** [review]
LGTM! 🚀

Just a small thing I've seen. Please, have a look.

**Comment by bimiron** [comment]
Looks nice! 👍 The androidTest called ImageModeTest should also be removed since we are not using the MapImageView anymore

**Pull request merged**